src/config.rs

```diff
@@ -22,7 +22,11 @@ pub struct CornerConfig {
     #[serde(default = "default_locations")]
     pub locations: Vec<Location>,
     #[serde(default = "default_size")]
-    pub size: u8,
+    pub size: u16,
+    #[serde(default = "default_size")]
+    pub size_width: u16,
+    #[serde(default = "default_size")]
+    pub size_height: u16,
```

Comment:
Maybe we could introduce a type so that you can input `10,15` for `width[,height]` where `height` is optional and assumed to be the same as width if not set.

A custom type would also allow us to support more than just numbers. It might be interesting to have `fill` or `auto` as an option to automatically use all of the available height/width. For example if you use the location left and `fill` as height, it would allow you to specify an entire side of the screen. 🤔 What do you think?

Reply:
That certainly sounds like a better solution. I had it like this only really because it was the first thing I thought of.